modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiNodeLocalServiceImpl.java

```diff
@@ -570,6 +603,192 @@ private List<WikiNode> _addDefaultNode(long groupId)
 		return ListUtil.fromArray(node);
 	}
 
+	private String _getCreoleRedirectContent(String redirectTitle) {
```

Comment:
He @MrJohn1911, in this case, we are mixing up the `MediaWikiImporter` and the `WikiNodeLocalService` responsibilities. So, keep it as a separate class and inject the dependencies via constructor. Thx